### ISSUE TITLE
running setup doesn't take config overrides from environment into account

### ISSUE THREAD

**Issue by regevbr** [jhuckaby/Cronicle#182]
## Summary

Running `control.sh setup` doesn't take config overrides from environment into account.

## Steps to reproduce the problem

Let use env variables to move the storage engine to be S3 and you will notice that the setup phase doesn't create any files on the S3 bucket

Set up file-system storage in config.json and run

```bash
export CRONICLE_Storage__engine="S3"` 
/opt/cronicle/bin/control.sh setup
```
### Operating system and version?

Ubuntu 18

### Node.js version?

6.14

### Cronicle software version?

0.8.28

### Are you using a multi-server setup, or just a single server?

Single

### Are you using the filesystem as back-end storage, or S3/Couchbase?

S3

### Can you reproduce the crash consistently?

Yes

**Comment by regevbr** [comment]
Currently I had to re write the configuration into config.json before running the setup which is far from ideal...

**Comment by jhuckaby** [comment]
Fixed in [v0.8.29](https://github.com/jhuckaby/Cronicle/releases/tag/v0.8.29).  Thanks!

**Comment by regevbr** [comment]
@jhuckaby My man! Thanks for the quick handling :-)
